website/pages/docs/nia/installation/requirements.mdx

```diff
@@ -40,8 +42,104 @@ The modules listed below are availabe to use and are comptabible with consul-ter
 
 #### Palo Alto Networks
 
-### How to Create a Terraform Module
+### How to Create a Compatible Terraform Module
+
+-> **Note:** Consul-Terraform-Sync is currently in Technology Preview. Specifications in this section may change going into Beta release.
 
 You can read more on how to [create a module](https://www.terraform.io/docs/modules/index.html) or work through a [tutorial to build a module](https://learn.hashicorp.com/tutorials/terraform/module-create). Consul-Terraform-Sync is designed to integrate with any module that satisfies the specifications in the following section.
 
-## Run a Consul Cluster
+#### Module Specifications
```

Comment:
I'm totally going to link to this from that architecture doc's Drivers section... "It comes
with a set of providers that [enables supporting](/docs/nia/requirements#module_specification) a wide variety of infrastructure applications."